hopsworks-api/src/main/java/io/hops/hopsworks/api/kibana/KibanaProxyServlet.java

```diff
@@ -143,7 +144,9 @@ protected void service(HttpServletRequest servletRequest,
     } else if (servletRequest.getRequestURI().contains("elasticsearch/*/_search")) {
       kibanaFilter = KibanaFilter.ELASTICSEARCH_SEARCH;
     } else if (servletRequest.getRequestURI().contains("legacy_scroll_start")) {
```

Comment:
@robzor92 `legacy_scroll_start` and `settings/defaultIndex` can be in the same `if` statement since they return the same thing